index.js

```diff
@@ -180,21 +174,11 @@ class IgnitorPlugin {
   }
   
   wrapFunction(name) {
-    const config = this.sls.service.functions[name];
-    this.sls.cli.log(`Wrapped ${config.handler}`);
-
-    // paths are constructed like <path>.<module> 
-    const [relativePath, module] = config.handler.split('.');
-    const handlerPath = `${PACKAGE_DIR}/${name}.handler`;
-    const handlerFileName = `${name}.js`;
-    const wrappedFilePath = path.resolve(PACKAGE_DIR, handlerFileName);
-    const wrapperCode = template.generate(relativePath, module);
-
-    // write wrapped code to wrappedFilePath
-    utils.write(wrappedFilePath, wrapperCode);
+    const { handler } = this.sls.service.functions[name];
+    this.sls.cli.log(`Wrapped ${handler}`);
 
     // update handler path
-    this.sls.service.functions[name].handler = handlerPath;
+    this.sls.service.functions[name].handler = build.wrap(name, handler);
```

Comment:
offload this, it will be handled by the build util as it is generating content within the build directory.